src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    static final int COUNT_OF_FIGURES = 5;
+
+    public static void main(String[] args) {
+        FigureSupplier figuresupplier = new FigureSupplier();
```

Comment:
The variable name 'figuresupplier' does not follow Java naming conventions. Variable names should be written in camelCase. Consider renaming it to 'figureSupplier'.